alchy/model.py

```diff
@@ -126,12 +125,7 @@ class User(Model):
     @declared_attr
     def __table_args__(cls):  # pylint: disable=no-self-argument
         # pylint: disable=no-member
-        args, kargs = merge_declarative_args(cls.__bases__, '__table_args__')
```

Comment:
Is there any harm in continuing to merge `__table/mapper_args__` along with `local`? Otherwise, this seems like a breaking change as any previously defined mixins (applied post-declarative base with the "final" model classes) that set `__table/mapper_args__` will now be ignored.